### DISCUSSION TITLE
Please consider adding support for notes created in a specific day (so that we can see all notes we created in a specific day)

### DISCUSSION THREAD

**Discussion by ricardonevesmusic** [liamcain/obsidian-calendar-plugin#334] [Ideas]
Hi

Please consider adding support for notes created in a specific day (so that we can see all notes we created in a specific day)

I hope you can add support for this in the future.

It would be nice to be able to see/access multiple notes we created in a specific day instead of just one note (Daily Note) per day.

Thank you.

**Comment by mattmaiorana** [comment]
You can do this by adding a dataview to your daily note template that pulls in all notes created or modified that day.
